test/models/review_test.rb

```diff
@@ -1,7 +1,58 @@
 require "test_helper"
 
 describe Review do
```

Comment:
Needs quotes around `Review`
```suggestion
describe "Review" do
```